app/views/samples_reports/show.haml

```diff
@@ -131,6 +130,10 @@
                       showCloseButton: false,
                       message: "After downloading these files, you will need to fill specific columns. <br />Make sure you read the instructions to be aware of these considerations <br>contained in the file called Instructions.txt")
 
+= form_tag print_samples_reports_path, method: "post", id: "print-pdf-form", target: "_blank" do
+  %input{type: "hidden", name: "json", id: "print-pdf-json"}
+  %input{type: "hidden", name: "context", id: "print-pdf-context"}
```

Comment:
You can set the current context directly as the input's value (changing context will re-render the page, and update the form accordingly).